homeassistant/core.py

```diff
@@ -233,12 +202,22 @@ def async_add_job(self, target: Callable[..., None], *args: Any):
         target: target to call.
         args: parameters for method to call.
         """
+        task = None
+
+        @asyncio.coroutine
+        def _run_executor():
+            yield from self.loop.run_in_executor(None, target, *args)
+
         if is_callback(target):
             self.loop.call_soon(target, *args)
```

Comment:
I could make a coro wrapper but that should be kill the reason why this is not a coro is.